Vungle/VungleRouter.m

```diff
@@ -541,13 +546,25 @@ - (void)clearWaitingList
     for (id key in self.waitingListDict) {
         id<VungleRouterDelegate> delegateInstance = [self.waitingListDict objectForKey:key];
         
+        NSString *targetPlacementID = [delegateInstance getPlacementID];
+        BOOL needRequestAd = YES;
+        
+        // If a placement is requested as priority placement at init call, no need to request again.
+        if ([self.prioritizedPlacementID isEqualToString:targetPlacementID]) {
+            needRequestAd = NO;
+        }
+        
         if ([delegateInstance respondsToSelector:@selector(getBannerSize)]) {
             NSString *id = [delegateInstance getPlacementID];
             CGSize size = [delegateInstance getBannerSize];
-            [self requestBannerAdWithPlacementID:id size:size delegate:delegateInstance];
+            [self requestBannerAdWithPlacementID:id size:size delegate:delegateInstance needRequestAd:needRequestAd];
         } else {
             if (![self.delegatesDict objectForKey:key]) {
                 [self.delegatesDict setObject:delegateInstance forKey:key];
+                
+                if (!needRequestAd) {
+                    continue;
+                }
```

Comment:
Do we need to put 
`if (!needRequestAd) {
                    continue;
            } `
out of this block, if `delegatesDict` contains the key, it will still call loadPlacementWithID, thanks a lot. ^_^

Reply:
Good catch. Updated.